pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -51,9 +51,12 @@ buildDotnetModule rec {
     ln --force --symbolic "${lib.getExe _7zzWithOptionalUnfreeRarSupport}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
   '';
 
+  NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR = 1; # Use nixpkgs 7zz executable
```

Comment:
Does passing arbitrary attrs to `buildDotnetModule` put them in the env or should this line be moved elsewhere?

Reply:
According to [this](https://stackoverflow.com/a/27719330/96588), yes.